catalog/core/catalog-core-api/src/main/java/ddf/catalog/data/AttributeRegistry.java

```diff
@@ -54,5 +54,5 @@
      * @return an {@link Optional} containing the registered {@link AttributeDescriptor}
      * @throws IllegalArgumentException if {@code name} is null
      */
-    Optional<AttributeDescriptor> getAttributeDescriptor(String name);
+    Optional<AttributeDescriptor> lookup(String name);
```

Comment:
I wanted to highlight the interface changes here.  This is marked as an experimental interface so this is allowed, but I know there are already places in the Catalog UI this is being used @rzwiefel